tests/ctst/features/cloudserverAuth.feature

```diff
@@ -0,0 +1,56 @@
+Feature: AWS S3 Bucket operations
+
+    @2.6.0
+    @PreMerge
+    @Cloudserver-Auth
+    Scenario: Check Authentication on bucket object lock actions with Vault
+        Given a IAM_USER type
+        And an IAM policy attached to the entity "user" with "Allow" effect to perform "CreateBucket" on "*"
+        And an IAM policy attached to the entity "user" with "<allow>" effect to perform "PutBucketObjectLockConfiguration" on "*"
+        And an IAM policy attached to the entity "user" with "<allow>" effect to perform "PutBucketVersioning" on "*"
+        When the user tries to perform CreateBucket
+        Then it "<should>" pass Vault authentication
+
+        Examples:
+            | allow | should     |
+            | Allow | should     |
+        # TODO: reenable after fix CLOUDSERVER-401
+        #    | Deny  | should not |
+
+
+    @2.6.0
+    @PreMerge
+    @Cloudserver-Auth
+    Scenario: Check Authentication on bucket object lock actions with Vault
+        Given a IAM_USER type
+        And an IAM policy attached to the entity "user" with "Allow" effect to perform "PutObjectRetention" on "*"
+        And an IAM policy attached to the entity "user" with "<allow>" effect to perform "BypassGovernanceRetention" on "*"
+        And an existing bucket "" "" versioning, "with" ObjectLock "GOVERNANCE" retention mode
```

Comment:
Nit, I wonder if we could still pass meaningful strings here linke 'without', and handle it in the typescript implementation, what do you think? It's mainly to have logical sentences, because at some point these tests will be exposed to non-technical readers, that may wonder what that means :slightly_smiling_face: